packages/thirdweb/src/wallets/eip5792/send-calls.ts

```diff
@@ -176,6 +177,10 @@ export async function sendCalls<const ID extends WalletId>(
       capabilities,
       version,
       chainId: numberToHex(chain.id),
+
+      // `true` seems like a better default than false:
+      // see: https://eips.ethereum.org/EIPS/eip-5792#wallet_sendcalls
+      atomicRequired: options.atomicRequired ?? true,
```

Comment:
false is prob safer though, who knows what the wallet supports on the other side